resources/lang/fr/dashboard.php

```diff
@@ -2,13 +2,26 @@
 
 return array (
   'devices_logged' => 'devices logged',
-  'getting_started_header' => 'Getting Started',
-  'getting_started_text' => 'Welcome to the Restarters community space.  Here are a few actions to get you started:',
-  'add_avatar' => 'Add an avatar',
-  'add_skills' => 'Add your skills',
-  'join_group' => 'Join a group',
-  'rsvp_event' => 'RSVP to an event',
   'visit_wiki' => 'Visit the wiki',
```

Comment:
These 3 new lines (visit_wiki, discussion_header, join_discussion) should be included.  (We will ask a volunteer for translations for them.)